.travis.yml

```diff
@@ -15,17 +15,14 @@ cache:
     - $HOME/.composer/cache
 
 php:
-  - 7.0
+  - 7.1
```

Comment:
Let the package require `php-7.1` version at least. The reasons are as follows:

- The `php-7.0` has been unsupported in official PHP team.
- The array with variable assignment approach is available in `php-7.1+`.

For example, the code snippets are as follows:

![image](https://user-images.githubusercontent.com/9021747/59007744-61c14100-885a-11e9-8422-ba690e6e256f.png)

Reply:
Yes, that correct